src/customerInfo.ts

```diff
@@ -48,13 +48,25 @@ export interface PurchasesEntitlementInfo {
      *
      * @note: Entitlement may still be active even if user has unsubscribed. Check the `isActive` property.
      */
-    readonly unsubscribeDetectedAt: string | null;
+    readonly unsubscribedDetectedAtMillis: number | null;
+    /**
+     * The date in milliseconds an unsubscribe was detected. Can be `null`.
```

Comment:
These should be the docs for `unsubscribedDetectedAtMillis` instead right? And the other way around